cdserver/features/utilization/step_definitions/utilization_steps.rb

```diff
@@ -38,7 +36,7 @@
     building = ShelterBuilding.find_by(name: entry['Building'])
     Census.create!(
       count: entry['Occupied'],
-      observed_for: DateTime.parse(entry['Business Date']),
```

Comment:
please change business date to shelter date in the gherkin too
i think that is a better name
thx